templates/default/customer/customersearch.html

```diff
@@ -89,7 +89,7 @@ <H1>{$layout.pagetitle}</H1>
 	<TBODY>
 	<TR>
 		<TD class="bold nobr">
-			<IMG src="img/customer.gif" alt=""> {trans("First/last or Company name:")}
+			{icon name="customer" fw=true} {trans("First/last or Company name:")}
```

Comment:
w {icon możesz włożyć `label="First/last or Company name:"`

Reply:
Ja bym etykiety tekstowej nie łączyć z ikoną. Jeśli już to trzeba by przygotować nową funkcję Smarty powiedzmy `{label}`, która obsłużyłaby i ikonkę i etykietę tekstową. Ikona to ikona.

Reply:
Dlaczego lepiej zrobić oddzielną funkcję `{label}` aniżeli wrzucić to do `{icon` jako opcjonalny parametr?